compiler/rustc_mir_transform/src/check_packed_ref.rs

```diff
@@ -56,8 +56,9 @@ impl<'tcx> Visitor<'tcx> for PackedRefChecker<'_, 'tcx> {
                         "reference to packed field is unaligned"
                     )
                     .note(
```

Comment:
This should probably be broken up into several notes, e.g.

> packed structs are only aligned by one byte, and many modern architectures penalize unaligned field accesses

> creating a misaligned reference is undefined behavior (even if that ...

Reply:
I tried to phrase my suggestion to avoid these periods in the note, because they don't match the style of other notes. Can you rephrase this to be more like what I suggested here?

Reply:
Oh yeah, I totally missed that one. Apologies.